etc/release_notes/0.1.9.md

```diff
@@ -0,0 +1,67 @@
+Dear all,
+
+we are happy to announce the 0.1.9 release of `coq-lsp`.
```

Comment:
```suggestion
We are happy to announce the 0.1.9 release of `coq-lsp`.
```

Reply:
Thanks a lot @Alizter , all the comments should be incorporated.